### PR TITLE
Colours endpoints

### PR DESCRIPTION
**Pull request by jackcasstlesjones**
## Description

Created routes/endpoints:
- GET all colours
- GET one colour
- POST one colour

## Type of change

- [ ] Bug fix
- [x] New feature
- [ ] Refactoring
- [ ] Documentation update
- [ ] Other (please describe):

## How has this been tested?

Please describe the testing methods used.

🏃

## Checklist:

- [x] I have reviewed the code changes.
~~- I have tested the code changes.~~
~~- I have updated documentation accordingly.~~
~~- My changes do not break existing functionality.~~

## Additional comments

# PLS BE CRITICAL OF THIS PR AS IT COULD BE QUITE A MESS. IF YOU SEE ANYTHING WRONG DON'T APPROVE, LEAVE A REVIEW AND REQUEST CHANGES thanks xxx

### CODE REVIEW

File: Data/DbContext.cs [hunk 1]
```diff
@@ -1,13 +1,12 @@
+using J3.Models;
 using Microsoft.EntityFrameworkCore;
 
-namespace j3.Data
+namespace J3.Data
```
Comment (JoshCodedit) [inline Data/DbContext.cs:4]: Thank you for correcting this for me

File: Data/DbContext.cs
Comment (JasonWarrenUK) [inline Data/DbContext.cs:1]: I highly approve of all formatting edits in this file

File: routes/ColourRoutes.cs
Comment (JasonWarrenUK) [inline routes/ColourRoutes.cs:1]: Nice. I've read through the whole file and, despite my limited understanding, your comments were enough for me to sort of follow the logic and it seems sensible to me.

**Review by JasonWarrenUK — APPROVED** [review]
Nice. Comments were great, I was actually able to follow the flow of logic despite not being familiar witht he syntax.

**Review by JoshCodedit — APPROVED** [review]
Looks good. I agree the comments have saved me a lot of time getting my head around the model binding. If anything we can remove them later once we are all comfortable with the concepts

**Pull request merged**